meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -34,12 +34,16 @@ Panel {
     title: "Image Gallery"
     implicitWidth: (root.defaultCellSize + 2) * 2
 
+    onReadOnlyChanged: {
+        m.readOnly = (_reconstruction && m.currentCameraInit) ? (m.currentCameraInit.locked || displayHDR.checked) : (root.readOnly || displayHDR.checked)
+    }
+
     QtObject {
         id: m
         property variant currentCameraInit: _reconstruction && _reconstruction.tempCameraInit ? _reconstruction.tempCameraInit : root.cameraInit
         property variant viewpoints: currentCameraInit ? currentCameraInit.attribute('viewpoints').value : undefined
         property variant intrinsics: currentCameraInit ? currentCameraInit.attribute('intrinsics').value : undefined
-        property bool readOnly: root.readOnly || displayHDR.checked
+        property bool readOnly: (_reconstruction && currentCameraInit) ? (currentCameraInit.locked || displayHDR.checked) : (root.readOnly || displayHDR.checked)
```

Comment:
You can factor out the `displayHDR.checked` to avoid writing it twice

Reply:
Done.